src/components/Footer/Footer.jsx

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
-import { Box, Typography, Link } from '@mui/material';
-import './Footer.css';
+import { Typography, Link } from '@mui/material';
+import { FooterContainer, FooterLinks } from './Footer.styled.js';
+import { Link as RouterLink } from 'react-router-dom';
+import { ENDPOINTS as ROUTER_PATHS } from '../../constants/routerPaths.js';
 
 function Footer() {
-    return (
-        <Box component="footer" className="footer-container">
-            <Typography variant="body2" color="textSecondary" sx={{ mb: 1 }}>
-                © {new Date().getFullYear()} SecondHand Store
-            </Typography>
-            <Box className="footer-links">
-                <Link href="/about" underline="hover" color="inherit">About</Link>
-                <Link href="/terms" underline="hover" color="inherit">Terms</Link>
-                <Link href="/contact" underline="hover" color="inherit">Contact</Link>
-            </Box>
-        </Box>
-    );
+  return (
+    <FooterContainer component="footer">
+      <Typography variant="body2" color="textSecondary" sx={{ mb: 1 }}>
```

Comment:
This does not belong into the prettier pr, please only push the relevant prettier changes